core-bundle/src/EventListener/DataContainer/ArticleSlotListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\ArticleModel;
+use Contao\CoreBundle\DependencyInjection\Attribute\AsCallback;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\Twig\Inspector\Inspector;
+use Contao\DataContainer;
+use Contao\LayoutModel;
+use Contao\PageModel;
+
+class ArticleSlotListener
```

Comment:
The changes in this file I would extract into its own PR because this cleanup always makes sense, no matter this PR :) Of course without the `modern` adjustments.